software/artalk.yml

```diff
@@ -0,0 +1,13 @@
+name: "Artalk"
+website_url: "https://artalk.js.org/"
+source_code_url: "https://github.com/ArtalkJS/Artalk"
+description: "A self-hosted comment system built in Golang, providing a lightweight and highly customizable solution for adding comments to your website."
+licenses:
+  - MIT
+platforms:
+  - Go
+  - JavaScript
+  - Docker
```

Comment:
> https://artalk.js.org/guide/deploy.html

```suggestion
platforms:
  - Docker
  - Nodejs
```

Reply:
Hi there, since the server is written in Go (not Node.js) and is deployable via Docker, I believe it makes sense to keep it in Go. :)

Reply:
Only deployable platforms go in the platform section

Reply:
My deployment documentation wasn't clear enough before. It has now been updated. Artalk consists of both a **Server-side** and a **Client-side**. The server-side is a **Golang** program, which can be deployed via `go install github.com/artalkjs/artalk/v2@latest`, and the client-side is JavaScript code. Therefore, the platforms should include Golang.

Reply:
Hi, just checking in to see if you had a chance to review my last response. Thanks! :)

Reply:
As long as the docs are clarified now, It all looks good to me.

Reply:
Since we only display the languages of a software's server-side requirements, shouldn't we still remove JavaScript?